src/main/java/frc/robot/subsystems/climberSubsystem/ClimberSubsystem.java

```diff
@@ -53,8 +53,8 @@ private void configMotor(CANSparkMax motor, boolean invert) {
     pidController.setD(ClimberConstants.CURRENT_PID[2]);
     double armPosition;
 
-    armPosition = motor.getAbsoluteEncoder().getPosition();
```

Comment:
So we disabled the absolute encoder during competition? 
Was there a reason why? Are we not expecting to use the absolute encoder in district champs?

Reply:
it was occasionally placing the arms at large negative or positive values. Can be reimplemented when needed